js/fontfaceobserver.js

```diff
@@ -0,0 +1,16 @@
+// https://www.bramstein.com/writing/web-font-loading-patterns.html
+
+var font = new FontFaceObserver('Mali');
+
+var html = document.documentElement;
+
+html.classList.add('fonts-loading');
+
+font.load().then(function () {
+    console.log('Font is available');
+    html.classList.remove('fonts-loading');
+    html.classList.add('fonts-loaded');
+}).catch(function () {
+    html.classList.remove('fonts-loading');
+    html.classList.add('fonts-failed');
+});
```

Comment:
Am I missing where the FFO lib was added? I don't see it in this diff. Anyway, unless our FFO script has Promises polyfill, it will fail to function in IE - https://caniuse.com/#feat=promises

Reply:
Yeah.. it's complicated and often very specific to the use-case.

I actually think using `font-display: swap` is a great solution now that Chrome/FF/Safari all support it. [IE11/Edge behave that way by default](https://web.dev/fast/avoid-invisible-text). That's close to 100% of our required browser compat.

I took the FFO out of DSR recently because it was no longer bringing any benefit from my POV. Of course using Google Fonts complicates things, but we can also self-host the files and specify our own `font-display` that way.

Reply:
Some good news on this front: [Google Fonts now supports `font-display: swap` by default!](https://www.zachleat.com/web/google-fonts-display/)

That's a huge boon for implementing on other sites since it's just a small tweak to their existing Google Fonts calls if necessary (instead of pulling them down and self hosting)